### PR TITLE
Integrated Flask

### LINKED ISSUE #12: Integrate Flask

### PR DESCRIPTION
**Pull request by raresluncan**
Closes #12

### CODE REVIEW

File: templates/details.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+{% extends "index.html" %}
```
Comment (lgnstn) [inline templates/details.html]: Create a `layout.html` file as you had in the tutorial which will have the common content for all the pages and both index and details will extend that file.

**Review by lgnstn — CHANGES_REQUESTED** [review]
Before doing anything please rebase (or merge) with the master branch because I've merged another PR and there are some conflicts

**Comment by raresluncan** [comment]
@lgnstn Updated

File: static/styles.css [hunk 2]
```diff
@@ -183,6 +189,7 @@ body {
   margin-top: 8px;
 }
 
+<<<<<<< HEAD:styles/styles.css
```
Comment (lgnstn) [inline static/styles.css]: It looks like you forgot to resolve conflicts. Please do it.

File: firmware.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+from flask import Flask, request, session, g, redirect, url_for, abort, render_template, flash
+
+
+app=Flask(__name__)
+app.config.from_object(__name__)
+
+@app.route('/')
+def show_home():
+   return render_template('index.html')
+@app.route('/details')
```
Comment (lgnstn) [inline firmware.py]: Add 2 empty lines between functions in the same module, so before this line and before line 9

File: static/styles.css [hunk 4]
```diff
@@ -5,7 +5,9 @@
   height: auto;
   font-family: 'Roboto', sans-serif;
 }
-
+html {
+  position: absolute;
+}
```
Comment (lgnstn) [inline static/styles.css]: Please add an empty line after this block

**Review by lgnstn — CHANGES_REQUESTED** [review]
Make sure every time you create a commit and push the changes that everything is very well tested.

**Comment by raresluncan** [comment]
@lgnstn The required changes are done! Please take a quick review before merging in order to check the indenting.

File: templates/details.html [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        <input type="radio" id="star4" name="rating" value="4" /><label for="star4" title="NB BRAH">4</label>
+        <input type="radio" id="star3" name="rating" value="3" /><label for="star3" title="SO-SO">3 </label>
+        <input type="radio" id="star2" name="rating" value="2" /><label for="star2" title="LAME">2 </label>
+        <input type="radio" id="star1" name="rating" value="1" /><label for="star1" title="HORRIBLE">1</label>
+      </fieldset>
+    </div>
+  </div>
+{% endblock %}
+
+{% block footer %}
```
Comment (lgnstn) [inline templates/details.html:115]: You don't have to add the blocks in child template if you don't add anything in them

**Review by lgnstn — APPROVED** [review]
👍

**Pull request merged**